configs/ci.yaml

```diff
@@ -1,19 +1,19 @@
-plugins:
-  - path: ground_truth_slam/
-  - path: offline_imu_cam/
-  - path: open_vins/
-  - path: pose_prediction/
-  - path: gldemo/
-  - path: timewarp_gl/
-  - path: debugview/
-  - path: audio_pipeline/
-  # - path: holgoram/
-  # - path: zed/
+plugin_groups:
+  - !include "rt_slam_plugins.yaml"
+  - !include "core_plugins.yaml"
+  - plugin_group:
+    - path: gldemo/
+    - path: ground_truth_slam/
+    - path: debugview/
 
 loader:
   name: tests
 
-data: /does_not_exist
-demo_data: /does_not_exist
+data:
+  subpath: mav0
+  relative_to:
+    archive_path:
+      download_url: 'http://robotics.ethz.ch/~asl-datasets/ijrr_euroc_mav_dataset/vicon_room1/V1_02_medium/V1_02_medium.zip'
```

Comment:
It would help readability to add a comment that describes how this would be parsed. The config scheme is a dense read.